www/vorj00/ukol_06/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="cs">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Long live Sparta!</title>
+    <link rel="stylesheet" href="assets/css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container">
+        <form class="form" id="js-form">
+            <table class="table">
```

Comment:
proc jsi pouzil table?

Reply:
Chtěl jsem tam dát vedle toho ještě popis pro inputy, v tu chvíli by tabularita dávala větší smysl, zapomněl jsem pak na to to tam přidat a už jsem rovnou styloval, udělám další PR, nicméně validní to pak je, nejde o layout, ale form (https://codereview.stackexchange.com/questions/156394/html-form-using-table-or-div)

Reply:
v praxi se table vubec nepouziva, komplikuje to vyvoj, to mi doslo az pozdeji, tak jenom rikam